.github/workflows/maven.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       matrix:
         os: [ubuntu-latest, windows-latest ]
         java: ['8', '11']
-        wildfly-version: ['wildfly2400', 'wildfly2500']
+        wildfly-version: ['wildfly2300']
```

Comment:
I think the wildfly2400 could be much better and wildfly2500 can be kept. We have a excluded test list in https://issues.redhat.com/browse/JBWS-4230

Reply:
This is not going to run on tag wildfly-2400 either.  The only wildfly-24 branch that has the elytron changes is origin/24.x

Reply:
what do you mean by wildfly 2400 tag ? Is it WFLY 24.0.0.Final ?

Reply:
In the wildfly project there are 3 tagged wfly-24 versions, 24.0.0.Beta1, 
24.0.0.Final, 24.0.1.Final.  None of these tagged versions contains the
elyron changes in module webservices or the version update to jbossws-spi.
Currently branch 24.x is the only one that contains these changes.  Its version
number is 24.0.2.Final-SNAPSHOT.

I have been unsuccessful changing this yml file to checkout the 24.x branch and
build the project so that jbossws-cxf can use it to run the tests.